examples/create-appointment/index.js

```diff
@@ -93,19 +94,19 @@ app.post('/', function (request, response) {
   // Create appointment:
   var options = {
     method: 'POST',
-    body: {
+    data: {
       appointmentTypeID: appointmentTypeID,
       datetime:          time,
       firstName:         body.firstName,
       lastName:          body.lastName,
       email:             body.email
     }
   };
-  return acuity.request('/appointments', options, function (err, res, appointment) {
+  return acuity.request('/appointments', options).then(function (appointmentR) {
     response.render('index.html', {
-      appointment: JSON.stringify(appointment, null, '  ')
+      appointment: JSON.stringify(appointmentR.data, null, '  ')
     });
-  });
+  }).catch(function(err){return console.error("Error appointments: "+err.error+" with message: "+err.message);})
```

Comment:
Somehow I'm hitting this with the log: `Error appointments: bad_request with message: Expected a JSON object.`

I haven't been able to get to the bottom of it yet, but this is another example it'd be nice to display errors in the browser for (instead of an endless spinner).  Don't worry much about this one though — I can fix it on the branch.